model/pod/db.go

```diff
@@ -121,9 +121,6 @@ func FindByNeedsTermination() ([]Pod, error) {
 				StatusKey: StatusStarting,
 				bsonutil.GetDottedKeyName(TimeInfoKey, TimeInfoStartingKey): bson.M{"$lte": staleCutoff},
 			},
-			{
-				StatusKey: StatusDecommissioned,
```

Comment:
We have to leave this condition in. It should be sufficient to add a condition to check that the current running task is empty.